specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-06-02-preview/fleets.json

```diff
@@ -229,6 +229,9 @@
           }
         },
         "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
```

Comment:
set this to azure-async-operation please

Reply:
actually, after looking at the RP code, this will work too

Reply:
I tried that, but it won't pass CI. The rules in CI strictly limit that only `location` can be set here.

Reply:
![image](https://user-images.githubusercontent.com/81607949/179435904-a7c776b6-7e35-4d25-8571-c562d3d25aea.png)